mimic/model/heat_objects.py

```diff
@@ -135,6 +135,8 @@ def creation_response_json(self, absolutize_url):
 
 
 stack_creation = EventDescription()
+stack_check = EventDescription()
```

Comment:
I don't like the fact that we have these new event descriptions with no explanation of what they mean or what their behavior signatures are.  However, I don't want to block on that, given that there's no clear existing convention for doing this, there are other events to document, and it's not clear how to document the control-plane endpoint associated with them.  I think probably `:var:` fields in the module docstring would do it.

So would you mind opening a new issue for documenting these event descriptions in the heat mimic?